components/automate-cs-ocid/habitat/hooks/run

```diff
@@ -7,15 +7,31 @@ cd {{pkgPathFor "chef/oc_id"}}/oc_id
 
 export RAILS_ENV="{{cfg.rails_env}}"
 export RACK_ENV="{{cfg.rack_env}}"
-export CHEF_SECRETS_DATA=$(cat config/private-chef-secrets.json)
 
 export DATABASE_URL="{{cfg.DATABASE_URL}}"
 
+OCID_CONFIG_FOLDER_PATH="{{pkgPathFor "chef/oc_id"}}/oc_id/config"
+
+# Set webui key from erchef component
 WEBUI_SRC_PATH="/hab/svc/automate-cs-oc-erchef/data/webui_priv.pem"
 WEBUI_KEY=`sed ':a;N;$!ba;s/\n/\\n/g' $WEBUI_SRC_PATH`
-PRIVATE_CHEF_SECRETS_PATH="{{pkgPathFor "chef/oc_id"}}/oc_id/config/private-chef-secrets.json"
+PRIVATE_CHEF_SECRETS_PATH="$OCID_CONFIG_FOLDER_PATH/private-chef-secrets.json"
 ruby /hab/svc/automate-cs-ocid/config/tasks/set_webui_key.rb "$WEBUI_KEY" "$PRIVATE_CHEF_SECRETS_PATH"
```

Comment:
Follow an argument less approach and read from the environment variables. Reference block below.